src/model.jl

```diff
@@ -520,14 +520,14 @@ function get_consumer_total_endowment(jm, m, c::ScalarConsumer)
     endowments = []
     for d in m._demands
         if get_full(d.consumer) == c
-            push!(endowments, :(
-                +($((:($(en.quantity) * 
-                $(en.commodity)) for en in d.endowments)...))
-            ))
+            push!(endowments, 
+                +(((eval(swap_our_param_with_val(en.quantity)) * 
+                swap_our_param_with_val(en.commodity)) for en in d.endowments)...)
+            )
```

Comment:
All these `swap_our_param_with_val` are incorrect, we should always use `tojump` for all of these cases.